### DISCUSSION TITLE
Is the hidden attribute of a post not conveyed to other clients?

### DISCUSSION THREAD

**Discussion by venteto** [bluesky-social/atproto#2326] [General Discussion]
I saw a post in one of my lists on the bsky.app web interface, then clicked through to hide that post.

I reloaded the deck.blue web interface (bypassing my browser cache as well) and noticed the post I had hidden in a different interface was still present.

Just wondering whether the hiding of a post is not part of the protocol itself but it merely client-dependent, and if hiding a post might become part of he protocol in the future so that it does not matter what client you are using.

**Comment by bnewbold** [comment]
To some degree, this will depend on how the other client works. The preferences are accessible between clients, but it isn't always easy to parse and implement them, and when updating preferences created by another client there is danger of clobbering details.

Another frequent issue is caching. Preferences are pulled on demand, and pulling frequently can be inefficient, so clients (correctly!) tend not to. It might help to do a full page refresh.

We know that the way private preferences work currently is difficult to work with, especially between different client implementations. We have a plan to improve this, making the private preferences more of a "key/value" thing, somewhat like repositories, but still being private. That work may still be a ways off, however.
